docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -184,6 +184,28 @@ There you can find links to upgrade notes for other versions too.
         +    - -c
         +    - config_file=/var/lib/postgresql/data/postgresql.conf
         ```
+- create configuration file for your crons in your project base ([#989](https://github.com/shopsys/shopsys/pull/989))
+    - create new file `src/ShopBundle/Resources/config/services/cron.yml` with following code:
+        ```
+        services:
+            _defaults:
+                autowire: true
+                autoconfigure: true
+                public: false
+
+        #   Example:
+        #   Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportCronModule:
+        #       tags:
+        #           - { name: shopsys.cron, hours: '*', minutes: '*' }
+        ```
+    - update `src/Shopsys/ShopBundle/Resources/config/services.yml`:
+        ```diff
+        imports:
+        -   - { resource: forms.yml }
```

Comment:
i think that you dont want to delete forms.yml

Reply:
No i want to remove `forms.yml` and another services, because I'm using wildcard for loading resources from services folder as you can see below (`services/*.yml`).

Reply:
```suggestion
          - { resource: forms.yml }
```